CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-12-2
+This release is rolling out to all regions - estimated time for completed roll out is 2021-12-2 for public cloud and 2021-12-9 for sovereign clouds.
```

Comment:
Roll out date should be 2021-12-09 for public cloud and 2021-12-16 for sovereign clouds

Reply:
Noted i ll amend it for both places.